Documentation/devel/distribution-point.md

```diff
@@ -0,0 +1,174 @@
+# Distribution points 
+
+A *distribution point* represents a method of fetching an image from an input string. This does not mandate an images type. Usually a distribution point can only provide one image type but some can provide different image types. For example:
+
+* Some *Docker* registries also provide *OCI images*.
+* *Rkt* can internally implement the Docker distribution to natively fetch a Docker/OCI image from a registry or an use an ACI converted by [docker2aci][docker2aci GH].
+
+Before distribution points, rkt used *ImageTypes*. These mapped a specifically formatted input string to things like the distribution, transport and image type. This information is hidden now since all images are appc ACIs.
+
+Distribution points are used as the primary expression of information in the different layers of rkt.  This includes fetching and referencing in a CAS/ref store.
+
+## Distribution Points types
+
+Distribution points are categorized as two types:
+
+* Indirect distribution points
+* Direct distribution points
+
+Direct distribution points provide the final information needed to fetch the image while indirect distribution points take some indirect steps (e.g. discovery) before getting the final image location.
```

Comment:
after sed replacements above, join these sentences to the one above to make a single graf.